drivers/unix/os_unix.cpp

```diff
@@ -280,7 +280,12 @@ Error OS_Unix::execute(const String &p_path, const List<String> &p_arguments, bo
 	ERR_FAIL_COND_V(pid < 0, ERR_CANT_FORK);
 
 	if (pid == 0) {
-		// is child
+		// Child process
+
+		// Disable inherited signal handlers
+		disable_crash_handler(); // Godot's crash handler, in case it handles SIGABRT in the future
```

Comment:
Do we get the crash handler back eventually, or is it disabled for good if you ever run `OS::execute`?

Reply:
Yes, this is only in the child process. I'm closing this though, since apparently it doesn't solve #17615 on OSX. @bruvzg proposes replacing the `abort()` with `raise(SIGKILL)`. Another option would be re-introducing the NSTask PR with the addition of code to search in $PATH.